.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -1,66 +1,67 @@
-name: "\U0001F41E Bug Report"
-description: Create a report to help us improve lemmy-ui
-title: "[Bug]: "
+name: "Bug report"
+description: Create a bug report to help us improve Lemmy-UI!
+title: ""
 labels: ["bug", "triage"]
 body:
   - type: markdown
     attributes:
       value: |
-        Found a bug? Please fill out the sections below. 👍
-        Thanks for taking the time to fill out this bug report!
-        For backend issues, use [lemmy](https://github.com/LemmyNet/lemmy/issues/new/choose)
+        Thanks for taking the time to help improve Lemmy-UI by reporting a bug!
   - type: checkboxes
     attributes:
       label: Requirements
-      description: Before you create a bug report please do the following.
+      description: Before you create a bug report, please carefully check the following –
       options:
-        - label: Is this a bug report? For questions or discussions use https://lemmy.ml/c/lemmy_support
+        - label: This is a bug report, and if not, I'd post to https://lemmy.ml/c/lemmy_support instead.
```

Comment:
Making it in the format of "I agree to...", which feels more impactful.

Reply:
I think "I will post" fits better than "I'd post".

Reply:
Actually, on reflection imperative case seems better, e.g. "please post to..."